esrally/client/asynchronous.py

```diff
@@ -56,6 +56,9 @@ def is_closing(self):
     def close(self):
         self.closed = True
 
+    def abort(self):
```

Comment:
@pquentin - this was added because the underlying aiohttp code calls `abort()` on the client's transport when `enable_cleanup_closed: true`.